src/main/java/org/zoo/Fishes.java

```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 
 public class Fishes extends EarthCreatures {
 
```

Comment:
Is the constructor necessary here with the category? I don't see it used anywhere

Reply:
constructor is removed